.devcontainer/Dockerfile

```diff
@@ -0,0 +1,34 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+
+FROM mcr.microsoft.com/dotnet/core/sdk:2.1.602
```

Comment:
If you target the new .NET Core 3.0+ SDK container it contains PowerShell and you won't need to install it.

Reply:
I agree but before that the main build needs to move .net core 3 (I vaguely remember the build to not work with it, but maybe that was an issue with one of the previews)

Reply:
Gooood point. Looks like that is the case: https://github.com/PowerShell/PowerShellEditorServices/pull/1095

Maybe we can try to get that figured out first before this goes in? My team put in work to get PowerShell inside the SDK container - I'd love to leverage that if possible as a form of dogfooding it.

Reply:
@rjmholt abandoned that PR so I'm ok with this going in as is. When we move to .NET SDK 3.0, we can revisit changing the container.